nrf: doc: Minor doc changes to multi-image (#1645)
NordicPlayground/fw-nrfconnect-nrf: doc/nrf/ug_multi_image.rst

   cmake -Dmcuboot_CONF_FILE=prj_a.conf -DCONF_FILE=app_prj.conf

> The command line that is used to create the child images can be extended by adding flags to the CMake variable ``EXTRA_MULTI_IMAGE_CMAKE_ARGS``.

Reviewer: ```suggestion
You can extend the CMake command that is used to create the child images by adding flags to the CMake variable ``EXTRA_MULTI_IMAGE_CMAKE_ARGS``.
```
